Lib/unittest/loader.py

```diff
@@ -503,15 +503,31 @@ def _makeLoader(prefix, sortUsing, suiteClass=None, testNamePatterns=None):
         loader.suiteClass = suiteClass
     return loader
 
+# Issue 5846: getTestCaseNames, makeSuite, and findTestCases are deprecated as
```

Comment:
It would be nice to add docstrings with explanation that that functions are deprecated.

Reply:
BTW, `_makeLoader` will be removed too, so move this comment few lines up.